.circleci/config.yml

```diff
@@ -29,10 +22,17 @@ install_system_deps: &install_system_deps
       apt update
       apt install -y unzip
 
-version: 2
+defaults: &defaults
+  working_directory: ~/repo
 
 jobs:
-  test_elixir_1_8_otp_21:
+  build_elixir_1_8_otp_22:
+    docker:
+      - image: erlang:22.0
+        environment:
+          ELIXIR_VERSION: 1.8.2-otp-22
+          LC_ALL: C.UTF-8
+          COVERALLS_REPO_TOKEN: sp50A1O1w2fIHf8c6ED6iMRQvtLCTJbeu
```

Comment:
Does every build report to coveralls and would need this token? Or just the highest version?

Reply:
Good catch. I missed that when I copy/pasted. This library actually isn't on Coveralls. However, to answer your question, on other projects, I've only had one build report to Coveralls.